pdal/StageExtensions.cpp

```diff
@@ -80,7 +80,9 @@ R"PDALEXTENSIONS(
     "writers.sqlite" : "sqlite, gpkg",
     "readers.mrsid" : "sid",
     "readers.rxp" : "rxp",
-    "readers.fbx" : "fbx"
+    "readers.fbx" : "fbx",
+    "readers.i3s" : "i3s",
```

Comment:
This isn't right, is it?  I thought I3S was supported only by protocol `i3s://`, and that only SLPK was supported by extension.  Is a `.i3s` path handled properly?

Reply:
Stagefactory portion that looks for protocols adds the .i3s extension to the file.